RELEASE/scripts/autoscend/auto_util.ash

```diff
@@ -1385,60 +1396,68 @@ boolean cloverUsageInit()
 	{
 		abort("Called cloverUsageInit but have no clovers");
 	}
-
+	int haveIntrinsicConstant = 2147483647; 
 	backupSetting("cloverProtectActive", false); // maybe set this before we return?
-	if(item_amount($item[Ten-Leaf Clover]) > 0)
+	//do we already have Lucky!?
+	if(have_effect($effect[Lucky!]) == haveIntrinsicConstant)
 	{
 		return true;
 	}
 
-	if(item_amount($item[Disassembled Clover]) > 0)
+	//use a clover if we have one in inventory or closet
+	if(item_amount($item[11-Leaf Clover]) < 1 && closet_amount($item[11-Leaf Clover]) > 0)
```

Comment:
don't take from closet, you can just call retrieve_item(1, $item[11-Leaf Clover]); and if one is available (and autoSatisfyWithCloset is enabled) it'll be put in inventory.